### PR TITLE
[Form] Form view parent

### PR DESCRIPTION
**Pull request by bhavin-nakrani**
| Q | A |
| --- | --- |
| Branch? | "master" for new features |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #18882 |
| License | MIT |
| Doc PR | reference to the documentation PR, if any |

### CODE REVIEW

File: src/Symfony/Component/Form/FormView.php [hunk 1]
```diff
@@ -159,4 +159,14 @@ public function count()
...
     }
+
+    /**
+     * Returns whether the given parent exists.
+     *
+     * @return bool
+     */
+    public function hasParent()
+    {
+        return (null !== $this->parent) ? true : false;
```
Comment (HeahDude) [inline src/Symfony/Component/Form/FormView.php]: The ternary is useless, just return `null !== $this->parent`

File: src/Symfony/Component/Form/Tests/FormViewTest.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+class FormViewTest extends \PHPUnit_Framework_TestCase
+{
+    private $formView;
+
+    protected function setUp()
+    {
+        $this->formView = $this->getMock('Symfony\Component\Form\FormView');
+    }
+
+    public function testHasParent()
```
Comment (HeahDude) [inline src/Symfony/Component/Form/Tests/FormViewTest.php]: I'm not sure we need a file to unit test this method only

**Comment by HeahDude** [comment]
The cs fixes in this PR are either needless or out of the scope here.

Status: Needs Work

File: src/Symfony/Component/Form/Tests/FormViewTest.php (hunk 2 above)
Comment (bhavin-nakrani) [inline src/Symfony/Component/Form/Tests/FormViewTest.php]: should i remove this testcase?

**Comment by HeahDude** [comment]
On second thought this may be a BC break: testing `form.parent` will return true if the view has a parent when one would expect to test if it has a child named "parent".

So I'm afraid we cannot merge this PR.

**Pull request closed without merging**